cypress-tests/cypress/e2e/configs/Payment/Hipay.js

```diff
@@ -0,0 +1,600 @@
+import { customerAcceptance } from "./Commons";
+
+const successfulNo3DSCardDetails = {
+  card_number: "4111111111111111",
+  card_exp_month: "06",
+  card_exp_year: "50",
+  card_holder_name: "joseph Doe",
+  card_cvc: "123",
+};
+const successful3DSCardDetails = {
+  card_number: "4000000000000002",
+  card_exp_month: "06",
+  card_exp_year: "50",
+  card_holder_name: "joseph Doe",
+  card_cvc: "123",
+};
+const singleUseMandateData = {
+  customer_acceptance: customerAcceptance,
+  mandate_type: {
+    single_use: {
+      amount: 8000,
+      currency: "EUR",
+    },
+  },
+};
+
+const multiUseMandateData = {
+  customer_acceptance: customerAcceptance,
+  mandate_type: {
+    multi_use: {
+      amount: 8000,
+      currency: "EUR",
+    },
+  },
+};
+export const connectorDetails = {
+  card_pm: {
+    PaymentIntent: {
+      Request: {
+        currency: "EUR",
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
+        billing: {
+          address: {
+            line1: "1467",
+            line2: "CA",
+            line3: "CA",
+            city: "Florence",
+            state: "Tuscany",
+            zip: "12345",
+            country: "IT",
+            first_name: "Max",
+            last_name: "Mustermann",
+          },
+          email: "mauro.morandi@nexi.it",
+          phone: {
+            number: "9123456789",
+            country_code: "+91",
+          },
```

Comment:
you can make this billing a const since it is repeated else where in the code below.